dev/com.ibm.ws.cdi.jee_fat/fat/src/com/ibm/ws/cdi/jee/binding/ManagedBeanBindingTest.java

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package com.ibm.ws.cdi.jee.binding;
+
+import static com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions.SERVER_ONLY;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.ws.cdi.jee.binding.app.ManagedBeanBindingTestServlet;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.EERepeatActions;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+
+@RunWith(FATRunner.class)
+public class ManagedBeanBindingTest {
+
+    public static final String APP_NAME = "ManagedBeanBinding";
+    public static final String SERVER_NAME = "cdi12BindingServer";
+
+    @TestServlet(contextRoot = APP_NAME, servlet = ManagedBeanBindingTestServlet.class)
+    @Server(SERVER_NAME)
+    public static LibertyServer server;
+
+    @ClassRule
+    public static RepeatTests r = EERepeatActions.repeat(SERVER_NAME, EERepeatActions.EE11, EERepeatActions.EE10, EERepeatActions.EE9, EERepeatActions.EE7);
```

Comment:
Is EE8 required here too?

Reply:
We have a few CDI tests which run on EE8 but most don't since EE8 and EE9 are identical apart from the Javax -> jakarta transformation.

Reply:
Thanks! That does makes sense. I'll approve it now